src/ophyd_async/epics/_backend/_p4p.py

```diff
@@ -141,6 +141,16 @@ def descriptor(self, source: str, value) -> Descriptor:
         return dict(source=source, dtype="object", shape=[])  # type: ignore
 
 
+class PvaStructureConverter(PvaConverter):
+
+    def value(self, value):
+        return value.get("pvi")
+
+    def descriptor(self, source: str, value) -> Descriptor:
+        # This is wrong, but defer until we know how to actually describe a structure
+        return dict(source=source, dtype="object", shape=[])  # type: ignore
```

Comment:
Please can we turn this into a generic `DictConverter`, then not do the `pvi` extraction here, but do it in the class?

We could also error when returning the descriptor to say it can't be described in scans at the moment